bisq-mobile.adoc

```diff
@@ -0,0 +1,84 @@
+= Bisq Mobile
+:imagesdir: images
+:!figure-caption:
+
+There are a couple of efforts underway to enable mobile functionality for Bisq. One of those efforts is Bisq mobile notifications, and it's ready to use today.
+
+== Mobile notifications
+
+The Bisq mobile notifications apps let you get notifications from Bisq desktop on your iPhone or Android device. This doc explains how to set it up and how it works.
+
+=== 1. Get the app
+
+* iOS: _link coming soon—waiting for Apple approval_
+* Android: _link coming soon—waiting for Google approval_
```

Comment:
There is luckily no approval from Google required but currently we need a bit more testing so there is only a Beta version and very soon it will be publicly accessible. At the download page we use:
"Currently in Beta <br>(Join <a href="https://bisq.network/slack-invite">#bisq-mobile on Slack</a> for access)"